pkg/api/security/cert/cert_getter_test.go

```diff
@@ -0,0 +1,139 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package cert
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	configmock "github.com/DataDog/datadog-agent/pkg/config/mock"
+	"github.com/DataDog/datadog-agent/pkg/config/model"
+)
+
+func initMockConf(t *testing.T) (model.Config, string) {
+	testDir := t.TempDir()
+
+	f, err := os.CreateTemp(testDir, "fake-datadog-yaml-")
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		f.Close()
+	})
```

Comment:
```suggestion
	t.Cleanup(f.Close)
```

Reply:
since f.Close return an error it's not possible to pass it as is.

Reply:
Oops, my mistake.